keybindings.js

```diff
@@ -254,23 +242,10 @@ function byId(mutterId) {
     return actionIdMap[mutterId];
 }
 
-/**
- * Adapts an paperwm action handler to mutter's keybinding handler signature
- *
- * Works for 3.28 and 3.30
- */
-var asKeyHandler;
-if (global.screen) {
-    asKeyHandler = (actionHandler) =>
-        (display, screen, mw, binding) => {
-            return actionHandler(mw, Tiling.spaces.selectedSpace, {display, screen, binding});
-        };
-} else {
-    asKeyHandler = (actionHandler) =>
-        (display, mw, binding) => {
-            return actionHandler(mw, Tiling.spaces.selectedSpace, {display, binding});
-        };
-}
+var asKeyHandler = (actionHandler) =>
+    (display, mw, binding) => {
+        return actionHandler(mw, Tiling.spaces.selectedSpace, { display, binding });
+    };
```

Comment:
I think there is no point keeping this an arrow function since it is on the global scope anyway.